src/pages/MyMove/PPMBooking/Advance/Advance.test.jsx

```diff
@@ -50,6 +51,18 @@ const mockMTOShipment = {
   eTag: btoa(new Date()),
```

Comment:
Nit: This isn't anything you changed, but it looks like btoa is deprecated. You could replace it with the following
```suggestion
  eTag: Buffer.from(new Date()).toString('base64'),
```